### PR TITLE
L2D15A3

### PR DESCRIPTION
**Pull request by evyryan**
Added question headers.
Fixed fatheads.

### CODE REVIEW

File: kzz7yh-f46637/cod-ve09v2_kzz7yh-f46637.xml [hunk 1]
```diff
@@ -98,7 +99,7 @@
       na<lb ed="#V" n="82" break="no"/>ture est animal quam anima. 
    </p>
                <p xml:id="kzz7yh-f46637-d1e177">
-                  <lb ed="#V" n="83"/>Contra secundum philosophum. 5 metapiheum. Accidens est prius 
+                  <lb ed="#V" n="83"/>Contra secundum philosophum. 5 metapiheum<!--abbreviation-->. Accidens est prius
```
Comment (jeffreycwitt) [inline kzz7yh-f46637/cod-ve09v2_kzz7yh-f46637.xml:102]:
```suggestion
                  <lb ed="#V" n="83"/>Contra secundum <ref>philosophum. 5 metaph.</ref> Accidens est prius 
```

File: kzz7yh-f46637/cod-ve09v2_kzz7yh-f46637.xml [hunk 2]
```diff
@@ -201,12 +202,13 @@
...
             <div xml:id="kzz7yh-f46637-Dd1e380">
                <head xml:id="kzz7yh-f46637-Hd1e382">Quaestio 2</head>
+               <head type="question-title">utrum anima sit nobilior quam animal</head>
                <p xml:id="kzz7yh-f46637-d1e384">
       Quaesio II. 
       <lb ed="#V" n="26"/>SEcundo queritur vtrum anima sit 
       no<lb ed="#V" n="27" break="no"/>bilior: quam animal. Et videtur 
-      <lb ed="#V" n="28"/>quod sic Commen. in expositione verbi Arum. 7. meta. 
-      Si<lb ed="#V" n="29" break="no"/>species maeria est prior: et magis ens: et in ipso: quod ex 
+      <lb ed="#V" n="28"/>quod sic Commen. in expositione verbi Arum. 7. meta. Si
```
Comment (jeffreycwitt) [inline kzz7yh-f46637/cod-ve09v2_kzz7yh-f46637.xml:210]:
```suggestion
      <lb ed="#V" n="28"/>quod sic <ref>Commentator in expositione verbi Aristotelis. 7. metaphysicae</ref>. Si
```

File: kzz7yh-f46637/cod-ve09v2_kzz7yh-f46637.xml [hunk 3]
```diff
@@ -290,8 +292,8 @@
       <lb ed="#V" n="89"/>ostendunt rationes ad partem secundam. 
     </p>
                <p xml:id="kzz7yh-f46637-d1e550">
-                  <lb ed="#V" n="90"/>Ad primum n oppositum dicendum: quod 
-      for<lb ed="#V" n="91" break="no"/>inte Commentator: et etiam 
+                  <lb ed="#V" n="90"/>Ad primum n<!--n?--> oppositum dicendum: quod
```
Comment (jeffreycwitt) [inline kzz7yh-f46637/cod-ve09v2_kzz7yh-f46637.xml:295]:
```suggestion
                  <lb ed="#V" n="90"/>Ad primum <unclear>in<!-- "i" looks smudged away</unclear> oppositum dicendum: quod 
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]

**Comment by evyryan** [comment]
Why does this have an X by it???

**Comment by evyryan** [comment]
> Why does this have an X by it???

@jeffreycwitt

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**